Dockerfile

```diff
@@ -409,7 +412,14 @@ FROM build AS initramfs-archive-arm64
 WORKDIR /initramfs
 COPY --from=squashfs-arm64 /rootfs.sqsh .
 COPY --from=init-build-arm64 /init .
-RUN set -o pipefail && find . 2>/dev/null | cpio -H newc -o | xz -v -C crc32 -0 -e -T 0 -z >/initramfs.xz
+RUN find . -print0 \
```

Comment:
Do we need apply same for amd64 variant?

Reply:
Yeah this was just introduced with the rebase against master for me.

Also Found a one line diff regression as a result of that as well.

Grabbing both. Don't merge quite yet!